csbpg/public_schema.go

```diff
@@ -0,0 +1,22 @@
+package csbpg
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/lib/pq"
+)
+
+func relaxPublicSchemaRestrictions(tx *sql.Tx, cf connectionFactory) error {
```

Comment:
```suggestion
func grantAllPrivilegesToPublicSchema(tx *sql.Tx, cf connectionFactory) error {
```

I think the current name will make sense only now that we are solving this problem

Reply:
Thanks!